src/Language/PureScript/Ide/Externs.hs

```diff
@@ -95,6 +95,7 @@ convertDecl P.EDValue{..} = Just $ IdeDeclValue $
   IdeValue edValueName edValueType
 convertDecl P.EDClass{..} = Just (IdeDeclTypeClass edClassName)
 convertDecl P.EDInstance{} = Nothing
+convertDecl P.EDKind{} = Nothing
```

Comment:
I wasn't sure what this was.  I'll investigate further to see what actually needs doing here.  I assume I'll need to add a new `IdeDeclaration` constructor.